src/coaster_gui.cpp

```diff
@@ -524,17 +524,17 @@ void CoasterInstanceWindow::OnClick(WidgetNumber widget, const Point16 &pos)
 
 		case CIW_CHOOSE_ENTRANCE: {
 			DropdownList itemlist;
-			for (int i = 0; _rides_manager.entrances[i] != nullptr; i++) {
-				_str_params.SetUint8(1, _language.GetText(_rides_manager.entrances[i]->name));
+			for (auto &eetype : _rides_manager.entrances) {
```

Comment:
We got the option to use `const` data here, is that something to aim for? (I'd say yes, but maybe you have other plans?)

Reply:
You seem to use `const` at other places, so this one is likely a bad suggestion.

Reply:
As a rule I try to use `const` wherever possible – it's a good way to discover bugs already at compile-time. Only variables that are actually meant to be modified should be non-`const`.